scripts/basic_setup.yaml

```diff
@@ -38,17 +38,17 @@ forward_functions:
     lambda_torsions: lambda
 
 #The number of equilibrium steps to take between nonequilibrium switching events
-n_equilibrium_steps_per_iteration: 10000
+n_equilibrium_steps_per_iteration: 100
 
 #The length of the ncmc protocol
-n_steps_ncmc_protocol: 50000
+n_steps_ncmc_protocol: 50
 
 #The number of NCMC steps per move application. This controls the output frequency
 #1 step/move application means writing out the work at every step.
 n_steps_per_move_application: 1
 
 #where to put the trajectories
-trajectory_directory: /Users/grinawap/solvent_test
+trajectory_directory: /Users/grinawap/solvent_test_4
```

Comment:
We should not be using hardcoded absolute paths.

If `dask` needs absolute paths, it can check the current working directory from the submitting client and prepend this to all specified relative paths to generate absolute paths.